gobblin-core/src/main/java/gobblin/writer/http/AsyncHttpWriterBuilder.java

```diff
@@ -59,6 +62,8 @@
   protected int queueCapacity = AbstractAsyncDataWriter.DEFAULT_BUFFER_CAPACITY;
   @Getter
   protected int maxAttempts = AsyncHttpWriter.DEFAULT_MAX_ATTEMPTS;
+  @Getter
+  protected SharedResourcesBroker<GobblinScopeTypes> broker = null;
```

Comment:
These variables are not initialized on `init`, which doesn't provide the `state` information. They are initialized on `writeTo(Destination)`